packages/server/test/fixtures/lh-11-3-0-coursehero-b.json

```diff
@@ -0,0 +1,62 @@
+    }
+  },
+  "audits": {
+    "is-on-https": {
+      "id": "is-on-https",
+      "title": "Uses HTTPS",
+      "description": "All sites should be protected with HTTPS, even ones that don't handle sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), where some resources are loaded over HTTP despite the initial request being served over HTTPS. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users, and is a prerequisite for HTTP/2 and many new web platform APIs. [Learn more about HTTPS](https://developer.chrome.com/docs/lighthouse/pwa/is-on-https/).",
+      "score": 1,
+      "scoreDisplayMode": "binary",
+      "details": {
+        "type": "table",
+        "headings": [],
+        "items": []
+      }
+    },
+    "viewport": {
+      "id": "viewport",
+      "title": "Has a `<meta name=\"viewport\">` tag with `width` or `initial-scale`",
+      "description": "A `<meta name=\"viewport\">` not only optimizes your app for mobile screen sizes, but also prevents [a 300 millisecond delay to user input](https://developer.chrome.com/blog/300ms-tap-delay-gone-away/). [Learn more about using the viewport meta tag](https://developer.chrome.com/docs/lighthouse/pwa/viewport/).",
+      "score": 1,
+      "scoreDisplayMode": "metricSavings",
+      "warnings": [],
+      "metricSavings": {
+        "INP": 0
+      },
+      "guidanceLevel": 3
+    },
+    "first-contentful-paint": {
+      "id": "first-contentful-paint",
+      "title": "First Contentful Paint",
+      "description": "First Contentful Paint marks the time at which the first text or image is painted. [Learn more about the First Contentful Paint metric](https://developer.chrome.com/docs/lighthouse/performance/first-contentful-paint/).",
+      "score": null,
+      "scoreDisplayMode": "error",
+      "errorMessage": "The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)",
+      "errorStack": "LighthouseError: NO_FCP\n    at LHTraceProcessor.createNoFirstContentfulPaintError (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/lh-trace-processor.js:41:12)\n    at LHTraceProcessor.computeNavigationTimingsForFrame (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/tracehouse/trace-processor.js:957:18)\n    at LHTraceProcessor.processNavigation (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/tracehouse/trace-processor.js:800:31)\n    at ProcessedNavigation.compute_ (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/computed/processed-navigation.js:32:29)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
+    },
+    "largest-contentful-paint": {
+      "id": "largest-contentful-paint",
+      "title": "Largest Contentful Paint"
+    }
+  }
```

Comment:
This also didn't finish properly, which is the primary reason this pull request is in a draft state.

Reply:
```
...
Error: Lighthouse failed with exit code 1
    at ChildProcess.<anonymous> (/home/hamir/lighthouse-ci/packages/cli/src/collect/node-runner.js:120:21)
    at ChildProcess.emit (node:events:517:28)
    at ChildProcess._handle.onexit (node:internal/child_process:292:12)
{
  "lighthouseVersion": "11.3.0",
  "requestedUrl": "https://www.coursehero.com/",
  "mainDocumentUrl": "https://www.coursehero.com/",
  "finalDisplayedUrl": "https://www.coursehero.com/",
  "finalUrl": "https://www.coursehero.com/",
  "fetchTime": "2023-11-03T09:51:46.521Z",
  "gatherMode": "navigation",
  "runtimeError": {
    "code": "NO_FCP",
    "message": "The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)"
  },
  "runWarnings": [],
  "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/119.0.6045.105 Safari/537.36",
  "environment": {
    "networkUserAgent": "Mozilla/5.0 (Linux; Android 11; moto g power (2022)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Mobile Safari/537.36",
    "hostUserAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/119.0.6045.105 Safari/537.36",
    "benchmarkIndex": 1993.5,
    "credits": {
      "axe-core": "4.8.2"
    }
  },
  "audits": {
    "is-on-https": {
      "id": "is-on-https",
      "title": "Uses HTTPS",
      "description": "All sites should be protected with HTTPS, even ones that don't handle sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/pr
event-mixed-content/what-is-mixed-content), where some resources are loaded over HTTP despite the initial request being served over HTTPS. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users, and is a prerequisite for HTTP/2 and many new web platform APIs. [Learn more about HTTPS](https://developer.chrome.com/docs/lighthouse/pwa/is-on-https/).",                                                                                                                                                                                                         "score": 1,
      "scoreDisplayMode": "binary",
      "details": {
        "type": "table",
        "headings": [],
        "items": []
      }
    },
    "viewport": {
      "id": "viewport",
      "title": "Has a `<meta name=\"viewport\">` tag with `width` or `initial-scale`",
      "description": "A `<meta name=\"viewport\">` not only optimizes your app for mobile screen sizes, but also prevents [a 300 millisecond delay to user input](https://developer.chrome.com/blog/300ms-tap
-delay-gone-away/). [Learn more about using the viewport meta tag](https://developer.chrome.com/docs/lighthouse/pwa/viewport/).",                                                                                  "score": 1,
      "scoreDisplayMode": "metricSavings",
      "warnings": [],
      "metricSavings": {
        "INP": 0
      },
      "guidanceLevel": 3
    },
    "first-contentful-paint": {
      "id": "first-contentful-paint",
      "title": "First Contentful Paint",
      "description": "First Contentful Paint marks the time at which the first text or image is painted. [Learn more about the First Contentful Paint metric](https://developer.chrome.com/docs/lighthouse/pe
rformance/first-contentful-paint/).",                                                                                                                                                                              "score": null,
      "scoreDisplayMode": "error",
      "errorMessage": "The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)",
      "errorStack": "LighthouseError: NO_FCP\n    at LHTraceProcessor.createNoFirstContentfulPaintError (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/lh-trace-processor.js:41:12)\n    
at LHTraceProcessor.computeNavigationTimingsForFrame (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/tracehouse/trace-processor.js:957:18)\n    at LHTraceProcessor.processNavigation (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/lib/tracehouse/trace-processor.js:800:31)\n    at ProcessedNavigation.compute_ (file:///home/hamir/lighthouse-ci/node_modules/lighthouse/core/computed/processed-navigation.js:32:29)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"                                                                                          },
    "largest-contentful-paint": {
      "id": "largest-contentful-paint",
      "title": "Largest Contentful Paint",
     
Fri, 03 Nov 2023 09:51:44 GMT LH:ChromeLauncher Waiting for browser.
Fri, 03 Nov 2023 09:51:44 GMT LH:ChromeLauncher Waiting for browser...
Fri, 03 Nov 2023 09:51:44 GMT LH:ChromeLauncher Waiting for browser.....
Fri, 03 Nov 2023 09:51:44 GMT LH:ChromeLauncher Waiting for browser.....✓
Fri, 03 Nov 2023 09:51:45 GMT LH:status Connecting to browser
Fri, 03 Nov 2023 09:51:45 GMT LH:status Navigating to about:blank
Fri, 03 Nov 2023 09:51:45 GMT LH:status Benchmarking machine
Fri, 03 Nov 2023 09:51:46 GMT LH:status Preparing target for navigation mode
Fri, 03 Nov 2023 09:51:46 GMT LH:status Cleaning origin data
Fri, 03 Nov 2023 09:51:46 GMT LH:status Cleaning browser cache
Fri, 03 Nov 2023 09:51:46 GMT LH:status Preparing network conditions
Fri, 03 Nov 2023 09:51:46 GMT LH:status Navigating to https://www.coursehero.com/
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: DevtoolsLog
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: Trace
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: DevtoolsLog
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: Trace
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: Accessibility
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: AnchorElements
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: ConsoleMessages
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: CSSUsage
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: Doctype
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: DOMStats
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: EmbeddedContent
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: FontSize
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: Inputs
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: GlobalListeners
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: ImageElements
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: InstallabilityErrors
Fri, 03 Nov 2023 09:51:51 GMT LH:status Get webapp installability errors
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: InspectorIssues
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: JsUsage
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: LinkElements
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: MainDocumentContent
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: MetaElements
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: NetworkUserAgent
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: OptimizedImages
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: ResponseCompression
Fri, 03 Nov 2023 09:51:51 GMT LH:status Getting artifact: RobotsTxt
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: Scripts
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: SourceMaps
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: Stacks
Fri, 03 Nov 2023 09:51:52 GMT LH:status Collect stacks
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: TagsBlockingFirstPaint
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: TapTargets
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: TraceElements
Fri, 03 Nov 2023 09:51:52 GMT LH:TraceElements:error NO_FCP
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: ViewportDimensions
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: WebAppManifest
Fri, 03 Nov 2023 09:51:52 GMT LH:status Get webapp manifest
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: devtoolsLogs
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: traces
Fri, 03 Nov 2023 09:51:52 GMT LH:status Getting artifact: FullPageScreenshot
Fri, 03 Nov 2023 09:51:53 GMT LH:status Getting artifact: BFCacheFailures
Fri, 03 Nov 2023 09:51:53 GMT LH:status Analyzing and running audits...
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Uses HTTPS
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Has a `<meta name="viewport">` tag with `width` or `initial-scale`
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: First Contentful Paint
Fri, 03 Nov 2023 09:51:53 GMT LH:first-contentful-paint:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Largest Contentful Paint
Fri, 03 Nov 2023 09:51:53 GMT LH:largest-contentful-paint:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: First Meaningful Paint
Fri, 03 Nov 2023 09:51:53 GMT LH:first-meaningful-paint:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Speed Index
Fri, 03 Nov 2023 09:51:53 GMT LH:speed-index:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Screenshot Thumbnails
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Final Screenshot
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Total Blocking Time
Fri, 03 Nov 2023 09:51:53 GMT LH:total-blocking-time:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Max Potential First Input Delay
Fri, 03 Nov 2023 09:51:53 GMT LH:max-potential-fid:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Cumulative Layout Shift
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: No browser errors logged to the console
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Initial server response time was short
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Time to Interactive
Fri, 03 Nov 2023 09:51:53 GMT LH:interactive:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: User Timing marks and measures
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoid chaining critical requests
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoid multiple page redirects
Fri, 03 Nov 2023 09:51:53 GMT LH:redirects:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Web app manifest and service worker meet the installability requirements
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Configured for a custom splash screen
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Sets a theme color for the address bar.
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Manifest has a maskable icon
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Content is sized correctly for the viewport
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Displays images with correct aspect ratio
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Serves images with appropriate resolution
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Fonts with `font-display: optional` are preloaded
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoids deprecated APIs
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Minimizes main-thread work
Fri, 03 Nov 2023 09:51:53 GMT LH:mainthread-work-breakdown:error NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: JavaScript execution time
Fri, 03 Nov 2023 09:51:53 GMT LH:bootup-time:error NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Preload key requests
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Preconnect to required origins
Fri, 03 Nov 2023 09:51:53 GMT LH:uses-rel-preconnect:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: All text remains visible during webfont loads
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Diagnostics
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Network Requests
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Network Round Trip Times
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Server Backend Latencies
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Tasks
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Metrics
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Performance budget
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Timing budget
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Minimize third-party usage
Fri, 03 Nov 2023 09:51:53 GMT LH:third-party-summary:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Lazy load third-party resources with facades
Fri, 03 Nov 2023 09:51:53 GMT LH:third-party-facades:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Largest Contentful Paint element
Fri, 03 Nov 2023 09:51:53 GMT LH:Runner:warn TraceElements gatherer, required by audit largest-contentful-paint-element, encountered an error: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Largest Contentful Paint image was not lazily loaded
Fri, 03 Nov 2023 09:51:53 GMT LH:Runner:warn TraceElements gatherer, required by audit lcp-lazy-loaded, encountered an error: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoid large layout shifts
Fri, 03 Nov 2023 09:51:53 GMT LH:Runner:warn TraceElements gatherer, required by audit layout-shift-elements, encountered an error: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoid long main-thread tasks
Fri, 03 Nov 2023 09:51:53 GMT LH:long-tasks:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoids `unload` event listeners
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Avoid non-composited animations
Fri, 03 Nov 2023 09:51:53 GMT LH:Runner:warn TraceElements gatherer, required by audit non-composited-animations, encountered an error: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Image elements have explicit `width` and `height`
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Page has valid source maps
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Preload Largest Contentful Paint image
Fri, 03 Nov 2023 09:51:53 GMT LH:Runner:warn TraceElements gatherer, required by audit prioritize-lcp-image, encountered an error: NO_FCP
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Ensure CSP is effective against XSS attacks
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Script Treemap Data
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Site works cross-browser
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Page transitions don't feel like they block on the network
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Each page has a URL
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: `[accesskey]` values are unique
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: `[aria-*]` attributes match their roles
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Values assigned to `role=""` are valid ARIA roles.
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: `button`, `link`, and `menuitem` elements have accessible names
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: Elements with `role="dialog"` or `role="alertdialog"` have accessible names.
Fri, 03 Nov 2023 09:51:53 GMT LH:status Auditing: `[aria-hidden="true"]` is not present on the document `<body>`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[aria-hidden="true"]` elements do not contain focusable descendents
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA input fields have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA `meter` elements have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA `progressbar` elements have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[role]`s have all required `[aria-*]` attributes
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Elements with an ARIA `[role]` that require children to contain a specific `[role]` have all required children.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[role]`s are contained by their required parent element
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[role]` values are valid
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Elements with the `role=text` attribute do not have focusable descendents.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA toggle fields have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA `tooltip` elements have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA `treeitem` elements have accessible names
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[aria-*]` attributes have valid values
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[aria-*]` attributes are valid and not misspelled
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Buttons have an accessible name
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: The page contains a heading, skip link, or landmark region
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Background and foreground colors have a sufficient contrast ratio
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<dl>`'s contain only properly-ordered `<dt>` and `<dd>` groups, `<script>`, `<template>` or `<div>` elements.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Definition list items are wrapped in `<dl>` elements
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document has a `<title>` element
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[id]` attributes on active, focusable elements are unique
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: ARIA IDs are unique
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: All heading elements contain content.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: No form fields have multiple labels
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<frame>` or `<iframe>` elements have a title
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Heading elements appear in a sequentially-descending order
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<html>` element has a `[lang]` attribute
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<html>` element has a valid value for its `[lang]` attribute
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<html>` element has an `[xml:lang]` attribute with the same base language as the `[lang]` attribute.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Identical links have the same purpose.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Image elements have `[alt]` attributes
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Image elements do not have `[alt]` attributes that are redundant text.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Input buttons have discernible text.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<input type="image">` elements have `[alt]` text
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Elements with visible text labels have matching accessible names.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Form elements have associated labels
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document has a main landmark.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Links have a discernible name
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Links are distinguishable without relying on color.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Lists contain only `<li>` elements and script supporting elements (`<script>` and `<template>`).
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: List items (`<li>`) are contained within `<ul>`, `<ol>` or `<menu>` parent elements
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: The document does not use `<meta http-equiv="refresh">`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[user-scalable="no"]` is not used in the `<meta name="viewport">` element and the `[maximum-scale]` attribute is not less than 5.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<object>` elements have alternate text
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Select elements have associated label elements.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Skip links are focusable.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: No element has a `[tabindex]` value greater than 0
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Tables have different content in the summary attribute and `<caption>`.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Tables use `<caption>` instead of cells with the `[colspan]` attribute to indicate a caption.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Touch targets have sufficient size and spacing.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<td>` elements in a large `<table>` have one or more table headers.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Cells in a `<table>` element that use the `[headers]` attribute refer to table cells within the same table.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<th>` elements and elements with `[role="columnheader"/"rowheader"]` have data cells they describe.
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `[lang]` attributes have a valid value
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: `<video>` elements contain a `<track>` element with `[kind="captions"]`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Custom controls have associated labels
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Custom controls have ARIA roles
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: User focus is not accidentally trapped in a region
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Interactive controls are keyboard focusable
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Interactive elements indicate their purpose and state
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: The page has a logical tab order
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: The user's focus is directed to new content added to the page
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Offscreen content is hidden from assistive technology
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: HTML5 landmark elements are used to improve navigation
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Visual order on the page follows DOM order
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Uses efficient cache policy on static assets
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoids enormous network payloads
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Defer offscreen images
Fri, 03 Nov 2023 09:51:54 GMT LH:offscreen-images:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Eliminate render-blocking resources
Fri, 03 Nov 2023 09:51:54 GMT LH:render-blocking-resources:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Minify CSS
Fri, 03 Nov 2023 09:51:54 GMT LH:unminified-css:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Minify JavaScript
Fri, 03 Nov 2023 09:51:54 GMT LH:unminified-javascript:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Reduce unused CSS
Fri, 03 Nov 2023 09:51:54 GMT LH:unused-css-rules:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Reduce unused JavaScript
Fri, 03 Nov 2023 09:51:54 GMT LH:unused-javascript:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Serve images in next-gen formats
Fri, 03 Nov 2023 09:51:54 GMT LH:modern-image-formats:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Efficiently encode images
Fri, 03 Nov 2023 09:51:54 GMT LH:uses-optimized-images:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Enable text compression
Fri, 03 Nov 2023 09:51:54 GMT LH:uses-text-compression:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Properly size images
Fri, 03 Nov 2023 09:51:54 GMT LH:uses-responsive-images:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Use video formats for animated content
Fri, 03 Nov 2023 09:51:54 GMT LH:efficient-animated-content:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Remove duplicate modules in JavaScript bundles
Fri, 03 Nov 2023 09:51:54 GMT LH:duplicated-javascript:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoid serving legacy JavaScript to modern browsers
Fri, 03 Nov 2023 09:51:54 GMT LH:legacy-javascript:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Page has the HTML doctype
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Properly defines charset
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoids an excessive DOM size
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoids requesting the geolocation permission on page load
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: No issues in the `Issues` panel in Chrome Devtools
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoids `document.write()`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Detected JavaScript libraries
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Avoids requesting the notification permission on page load
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Allows users to paste into input fields
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Use HTTP/2
Fri, 03 Nov 2023 09:51:54 GMT LH:uses-http2:warn Caught exception: NO_FCP
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Uses passive listeners to improve scrolling performance
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document has a meta description
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Page has successful HTTP status code
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document uses legible font sizes
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Links have descriptive text
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Links are crawlable
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Page isn’t blocked from indexing
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: robots.txt is valid
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Tap targets are sized appropriately
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document has a valid `hreflang`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document avoids plugins
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Document has a valid `rel=canonical`
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Structured data is valid
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Page didn't prevent back/forward cache restoration
Fri, 03 Nov 2023 09:51:54 GMT LH:status Generating results...
Fri, 03 Nov 2023 09:51:54 GMT LH:ChromeLauncher Killing Chrome instance 14404
Runtime error encountered: The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)
error Command failed with exit code 1.                                       
```

Reply:
```
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Structured data is valid
Fri, 03 Nov 2023 09:51:54 GMT LH:status Auditing: Page didn't prevent back/forward cache restoration
Fri, 03 Nov 2023 09:51:54 GMT LH:status Generating results...
Fri, 03 Nov 2023 09:51:54 GMT LH:ChromeLauncher Killing Chrome instance 14404
Runtime error encountered: The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)
error Command failed with exit code 1.        
```

Reply:
Any suggestions on how to make this work?